data/core.yaml

```diff
@@ -346,6 +346,13 @@ en:
       not_downloaded:
         single: This feature can't be moved because parts of it have not yet been downloaded.
         multiple: These features can't be moved because parts of them have not yet been downloaded.
+    follow:
+      key: F
```

Comment:
On the one hand, it’s convenient that the <kbd>F</kbd> accelerator key matches the <ins>f</ins>ollow operation in Potlatch (miss you) and JOSM. On the other hand, we’re starting to run out of unmodified accelerator keys, especially ones that are reachable by the left hand.[^1] It’s forcing localizations that localize keys to play successive rounds of musical chairs. But that’s a broader issue and shouldn’t hold up this PR.

Perhaps a future enhancement could also assign the ⌘[ and ⌘] keys to this operation. It would be elegant to “zip” a way together with another way using the normal keyboard shortcuts for navigating along a way.

[^1]: If I remember correctly, some of the accelerator keys were chosen to be reachable by the left hand, assuming the right hand is operating a mouse in tandem. But I can’t remember where I saw that stated.